extension/source/QuillContext.tsx

```diff
@@ -66,9 +71,10 @@ export function QuillContextProvider({ children }: Props) {
 
 function QuillContextCells(
   storage: CellCollection,
+  encryptedStorage: CellCollection,
```

Comment:
Same changes as QuillController's constructor can be made here by importing the `StorageConfig` type since both arguments are part of same entity.